Sources/Testing/Parameterization/CustomTestArgumentEncodable.swift

```diff
@@ -14,6 +14,25 @@ private import Foundation
 
 /// A protocol for customizing how arguments passed to parameterized tests are
 /// encoded, which is used to match against when running specific arguments.
+///
+/// By default, the testing library checks whether a test argument conforms to
+/// `Encodable` and encodes it using `encode(to:)` if it does. `Encodable` is
+/// sufficient for many types, but for some types the output of that function
+/// may contain large data payloads that cause poor performance, may not be
+/// stable and unique, or may otherwise be deemed unsuitable for testing.
+///
+/// If the type of the argument does not conform to `Encodable` but it does
+/// conform to `Identifiable` and its associated `ID` type conforms to
+/// `Encodable`, the value of its `id` property is encoded with `encode(to:)`
+/// and is used as the encoded representation.
+///
+/// If neither of the approaches for encoded representation described above is
+/// sufficient, a type can be made to conform to
+/// ``CustomTestArgumentEncodable``. The testing library will then call
+/// ``encodeTestArgument(to:)`` on values of that type instead of `encode(to:)`.
```

Comment:
Does this symbol reference to `encodeTestArgument(to:)` need to be adjusted now, to prefix it with the protocol name, now that this documentation has been hoisted up a level?

Reply:
No, DocC is smart enough to understand that this comment is attached to `CustomTestArgumentEncodable` already. (It just works.)